docs/tutorials/slurm/compute_canada/compute_canada.rst

```diff
@@ -63,27 +61,25 @@ Then pull (download) the :program:`Apptainer/Singularity` container:
 
    $ apptainer pull miracl_latest.sif library://aiconslab/miracl/miracl:latest
 
-.. attention::
-
-   If you encouter the following error with the pull request:
-
-.. code-block:: 
+.. attention::  
 
-   FATAL:   Unable to get library client configuration: remote has no library client (see https://apptainer.org/docs/user/latest/endpoint.html#no-default-remote)
```

Comment:
Here it shows that the link is not indented, but when opening it codespaces it is properly formatted.

Reply:
Looks good in the rendered version!